model/implementation/DeliveryExecutionStateService.php

```diff
@@ -169,7 +173,6 @@ public function run(DeliveryExecution $deliveryExecution)
             );
         }
 
-        $this->setState($deliveryExecution, ProctoredDeliveryExecution::STATE_ACTIVE);
```

Comment:
Moving this one up to line [`156`](https://github.com/oat-sa/extension-tao-proctoring/pull/818/files#diff-df1444a5531c22723cfed8dd8c5746e2R156) is the fix.

Sorry for the rest of the changes in the scope, most of them were done automatically.